### DISCUSSION TITLE
Desenvolvimento para iPad ainda é uma demanda?

### DISCUSSION THREAD

**Discussion by barbier** [frontendbr/forum#1395] [Perguntas e Respostas]
Olá!
Recentemente apareceu uma oportunidade de adquirir um iPad 4 (2012), então andei pensando...
Desenvolver para tablets ainda é uma demanda do mercado ou as empresas caíram na questão de desenvolver para celulares e deixar a tela se adaptar sozinha?

**Comment by fabiochiquezi** [comment]
Vish tenho a mesma dúvida, parece que ninguém foca no desenvolvimento p/ tablet

**Comment by AssisrMatheus** [comment]
Eu acho que mesmo que não seja uma demanda, ainda está em demanda desenvolver para iOS, então se você não tem um iPhone, vale a pena para você ter exp de desenvolvimento no sistema da apple, que é um bocado diferente(pra gerar certificados, subir para apple store etc)

**Comment by adamfoerster** [comment]
Depende muito do mercado e tipo da empresa. O mais comum para empresas que vão usar tablets é usar Android (pois é muito mais barato). Geralmente são sistemas para tirar pedido em restaurante, fazer venda em algum tipo de varejo etc.
Para iPad acredito que a demanda seja mais para apps pagos a serem vendidos na App Store, como apps e jogos e geralmente a empresa vai mais pra esse lado de fazer para celular e ajustar para tablet.
Já nas considerações técnicas se quiser desenvolver PWAs o suporte no Safari do iOS é bem fraco. Não acho que valha a pena. 
Mas se teu objetivo for nativo ou sites com SPA é uma boa.

**Comment by didiraja** [comment]
Eu chutaria que não. Até no desenvolvimento web, iPad é tratado como uma resolução secundária. Tomando cuidado apenas pras coisas não quebrarem e pronto.

**Comment by simaodeveloper** [comment]
Quando falamos em desenvolvimento responsivo, estamos falando em criar uma aplicação que rode em todos os tamanhos de tela.

Claro que há o escopo de cada projeto, onde definimos os **targets** (features, browsers, dispositivos...).

A resposta que você quer na verdade depende do escopo dos projetos que você esta trabalhando, 
você vai ter que verificar se existe essa demanda no seu nicho, seus clientes pedem isso, os clientes deles usam este dispositivo?

Bom espero ter contribuído com meus 2 centavos.
